clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/provider/GoogleInfrastructureProvider.groovy

```diff
@@ -51,15 +52,27 @@ class GoogleInfrastructureProvider extends AgentSchedulerAware implements Search
   ]
 
   final Map<String, SearchableProvider.SearchResultHydrator> searchResultHydrators = [
-      (INSTANCES.ns): new InstanceSearchResultHydrator(),
-      (BACKEND_SERVICES.ns): new BackendServiceResultHydrator()
+    (BACKEND_SERVICES.ns): new BackendServiceResultHydrator(),
+    (HTTP_HEALTH_CHECKS.ns): new HttpHealthCheckResultHydrator(),
+    (INSTANCES.ns): new InstanceSearchResultHydrator()
   ]
 
   @Override
   Map<String, String> parseKey(String key) {
     return Keys.parse(key)
   }
 
+  private static class HttpHealthCheckResultHydrator implements SearchableProvider.SearchResultHydrator {
```

Comment:
Could sort the class definitions to match the order in which they are referenced in `searchResultHydrators`.